includes/settings/settings-page.php

```diff
@@ -263,7 +267,7 @@ final public function register_settings_fields() {
 						isset( $field['label'] ) ? $field['label'] : '',
 						$field['render'],
 						static::PAGE_ID,
-						$full_section_id,
+						( 'experiments' === $section_id && 'stable' !== $field['field_args']['release_status'] ) ? 'elementor-experiments-ongoing-section' : $full_section_id,
```

Comment:
Isn't it a generic page that should get its content from a dedicated file? (a file that is dedicated to the content of the experiment?).

Reply:
This is how WordPress works, I don't think we can add setting fields in a different way.
@kobizz What do you think?